apis/event/1.4/usage.md

```diff
@@ -0,0 +1,899 @@
+    "player_card_number":"",
+    "asset_number":"1234",
+    "host_id_player":"",
+    "idx":37656,
+    "cache_time":1565039283259,
+    "event_time":1562083037909,
+    "data": {
+        "CURRENT_CREDITS":19550,
+        "ELECTRONIC_REGULAR_CASHABLE_TRANSFERS_TO_GAMING_MACHINE_CREDITS":288000,
+        "ELECTRONIC_REGULAR_CASHABLE_TRANSFERS_TO_HOST_CREDITS":103600,
+        "GAMES_LOST":544,
+        "GAMES_PLAYED":627,
+        "GAMES_WON":83,
+        "IN_HOUSE_CASHABLE_TRANSFERS_TO_GAMING_MACHINE_CENTS":288000,
+        "IN_HOUSE_CASHABLE_TRANSFERS_TO_HOST_CENTS":103600,
+        "IN_HOUSE_TRANSFERS_TO_GAMING_MACHINE_THAT_INCLUDED_CASHABLE_AMOUNTS_QUANTITY":27,
+        "IN_HOUSE_TRANSFERS_TO_HOST_THAT_INCLUDED_CASHABLE_AMOUNTS_QUANTITY":20,
+        "MIBTime":1562083037909,
+        "REGULAR_CASHABLE_TICKET_IN_CENTS":19850,
+        "REGULAR_CASHABLE_TICKET_IN_QUANTITY":8,
+        "REGULAR_CASHABLE_TICKET_OUT_CENTS":159600,
+        "REGULAR_CASHABLE_TICKET_OUT_QUANTITY":21,
+        "SPECIAL_METERS_LEGACY_BONUS_DEDUCTIBLE_METER":159700,
+        "SPECIAL_METERS_POWER_RESET":129,"SPECIAL_METERS_SLOT_DOOR_OPEN":123,
+        "SPECIAL_METERS_TOTAL_CREDIT_VALUE_OF_BILLS":100,
+        "SPECIAL_METERS_TOTAL_DOLLAR_VALUE_OF_BILLS":1,
+        "TOTAL_CANCELLED_CREDITS":429150,
+        "TOTAL_CASHABLE_TICKET_IN_CREDITS":19850,
+        "TOTAL_CASHABLE_TICKET_OUT_CREDITS":159600,
+        "TOTAL_COIN_IN_CREDITS":31225,
+        "TOTAL_COIN_OUT_CREDITS":185075,
+        "TOTAL_CREDITS_FROM_BILLS_ACCEPTED":100,
+        "TOTAL_DROP_CREDITS":307950,
+        "TOTAL_ELECTRONIC_TRANSFERS_TO_GAMING_MACHINE_CREDITS":288000,
+        "TOTAL_ELECTRONIC_TRANSFERS_TO_HOST_CREDITS":103600,
+        "TOTAL_HAND_PAID_CANCELLED_CREDITS":165950,
+        "TOTAL_HAND_PAID_CREDITS":165950,
+        "TOTAL_MACHINE_PAID_EXTERNAL_BONUS_WIN_CREDITS":159700,
+        "TOTAL_MACHINE_PAID_PAYTABLE_WIN_CREDITS":25375,
+        "TOTAL_NUMBER_OF_1_BILLS_ACCEPTED":1,
+        "TOTAL_REGULAR_CASHABLE_TICKET_IN_CREDITS":19850,
+        "TOTAL_SAS_CASHABLE_TICKET_IN_CENTS":19850,
+        "TOTAL_SAS_CASHABLE_TICKET_IN_QUANTITY":8,
+        "TOTAL_SAS_CASHABLE_TICKET_OUT_CENTS":159600,
+        "TOTAL_SAS_CASHABLE_TICKET_OUT_QUANTITY":21,
+        "TOTAL_TICKET_IN_CREDITS":19850,
+        "TOTAL_TICKET_OUT_CREDITS":159600,
+        "TOTAL_WON_CREDITS":185075,
+        "meter_snap_flag":129,
+        "num_meters":41
+    }
+}
+```
+
+## Data Dictionary
+
+All packets other than `MachineInfo` and `PlayerInfo` contain the following fields
+
+| field | description |
+|---|-----|
+| type | The type of data carried in this packet. |
+| idx | Monotonically increasing uint64 value used to glohost order events in the Nex7 event stream |
+| cache_time | Time at which the global event collector received this event, represented as milliseconds since the start of the Unix epoch |
+| event_time | The time at which the Machine Interface Board (MIB) received this event |
```

Comment:
Add the same note about milliseconds since the start of the Unix epoch here as in the cache_time field